recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -34,7 +34,13 @@ CMDLINE_DEBUG ?= ""
 # CMDLINE_RNDIS = "modules-load=dwc2,g_ether g_ether.host_addr=<some MAC 
 # address> g_ether.dev_addr=<some MAC address>"
 # if the MAC addresses are omitted, random values will be used
-CMDLINE_RNDIS ?= ""
+CMDLINE_RNDIS ?= '${@oe.utils.conditional("ENABLE_RNDIS", "1", "modules-load=dwc2,g_ether", "", d)}'
+
+TMP_HOST_MAC_ADDR = '${@oe.utils.conditional('RNDIS_HOST_MAC_ADDR', None, "", " g_ether.host_addr=" + str(d.getVar('RNDIS_HOST_MAC_ADDR')), d)}'
+CMDLINE_RNDIS_HOST_MAC_ADDR ?= '${@oe.utils.conditional("ENABLE_RNDIS", "1", TMP_HOST_MAC_ADDR, "", d)}'
```

Comment:
Would it be cleaner to use an anon function to handle this? It gets a bit tangled. Same for the set below.

Reply:
Great point

Reply:
Done